collectors/30/kafka_offset_collector.py

```diff
@@ -66,26 +85,34 @@ def get_partitions(topic):
 def report():
     # see the monitoring.json chef role
     zk_quorums = os.getenv('MONITORED_ZOOKEEPER_QUORUMS')
-    kafka_brokers = os.getenv('KAFKA_BROKERS')
 
     if zk_quorums is None:
         raise RuntimeError('MONITORED_ZOOKEEPER_QUORUMS not found')
 
-    if kafka_brokers is None:
-        raise RuntimeError('KAFKA_BROKERS not found')
-
-    # we could just use ZK to find all topics, should we just do that?
-    topics = ['mobile_metrics', 'raw_events']
-
     consumer_group = 'secor_group'
 
     for zk_quorum in zk_quorums.split('|'):
         zk = KazooClient(hosts=zk_quorum)
-        kafka = KafkaClient(kafka_brokers)
-
         zk.start()
+
+        kafka_brokers = get_kafka_brokers(zk)
+        if kafka_brokers is None:
```

Comment:
I don't think this is possible; `get_children` always returns a list (or raises).